avalon/houdini/pipeline.py

```diff
@@ -20,7 +20,7 @@
 self._parent = None
 self._events = dict()
 
-AVALON_CONTAINERS = "AVALON_CONTAINERS"
+AVALON_CONTAINERS = "/obj/AVALON_CONTAINERS"
```

Comment:
This was a fix for where Houdini sometimes would do a "relative query" without an absolute path and would fail to resolve the container in some contexts.